.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,88 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Kombu
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  GITHUB_WORKFLOW: true
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macos-latest]
```

Comment:
macos & windows can run unit tests and container based integration can run on llinux only, so separate jobs should be created

Reply:
See this for a solution on MacOS:
https://github.com/actions/virtual-environments/issues/1143#issuecomment-652264388